File: nullbots/nullmage/SelectAlch.java
package nullbots.nullmage;

import com.runemate.game.api.osrs.local.hud.interfaces.Magic;
import com.runemate.game.api.script.framework.tree.LeafTask;

public class SelectAlch extends LeafTask {

    @Override
    public void execute() {
        Magic.HIGH_LEVEL_ALCHEMY.activate();
        NullMage.lastSpell = true;
    }
}

File: kotlin/archbots/archmage/treebot/leaves/SelectAlch.kt
package archbots.archmage.treebot.leaves

import archbots.nullmage.NullMage
import com.runemate.game.api.osrs.local.hud.interfaces.Magic
import archapi.treebot.framework.LeafTask

class SelectAlch : LeafTask() {

    override fun execute() {
        Magic.HIGH_LEVEL_ALCHEMY.activate()
        NullMage.lastSpell = true
    }
}
